client/src/shared/icons/menu.tsx

```diff
@@ -0,0 +1,24 @@
+import React from 'react';
+
+const MenuLogo: React.FC = () => {
```

Comment:
- add props for all icons. `height`, `width`, `className`. This is to make the icon flexible (apply to all related)